dev/tools/gen_defaults/lib/action_chip_template.dart

```diff
@@ -60,10 +64,10 @@ class _${blockName}DefaultsM3 extends ChipThemeData {
   Color? get surfaceTintColor => ${colorOrTransparent("$tokenGroup.container.surface-tint-layer.color")};
 
   @override
-  Color? get checkmarkColor => ${color("$tokenGroup.with-icon.selected.icon.color")};
+  Color? get checkmarkColor => null;
```

Comment:
Should we here use "$tokenGroup.with-icon.icon.color"?

Reply:
@QuncCccccc  I've written my thoughts on this in the main thread: https://github.com/flutter/flutter/pull/141701#issuecomment-1901332557
Let me know what you think